Documentation/check-crd-compat-table.sh

```diff
@@ -163,7 +163,7 @@ if ! git diff --quiet ${last_cilium_release}..${remote}/${release_version} $crd_
   if [[ "${current_release_version}" != "${expected_version}" ]]; then
     >&2 echo "Current version for branch ${release_version} should be ${expected_version}, not ${current_release_version}, please run the following command to fix it:"
     >&2 echo "git checkout ${remote}/${release_version} && \\"
-    >&2 echo "sed -i 's+${current_release_version}+${expected_version}+' $(get_line_of_schema_version ${release_version})"
+    >&2 echo "sed -i 's+${current_release_version}+${expected_version}+' $(get_line_of_schema_version ${release_version} | sed 's/\n/ /g')"
```

Comment:
https://stackoverflow.com/a/6677343

Reply:
Ah good point, I guess this is effective simply by passing the `get_line_of_schema_version` output through sed by itself, not because of the sed expression :+1: I can fix it up.

Reply:
The original problem here was that `get_line_of_schema_version` would paste a newline between each filename with a matching schema version, so that's the main newline to get rid of. It doesn't matter if there's a newline at the end.